.github/workflows/tests.yml

```diff
@@ -47,11 +47,18 @@ jobs:
       - name: Generate Xcode projects and compare against existing record
         run: |
           set -euo pipefail
+          bazlisk clean
           bazelisk query 'kind(xcodeproj, tests/macos/xcodeproj/...)' | xargs -n 1 bazelisk run
           bazelisk query 'attr(executable, 1, kind(genrule, tests/macos/xcodeproj/...))' | xargs -n 1 bazelisk run
           git diff --exit-code tests/macos/xcodeproj
+      - name: Ensure hmap files do not exist (or remove if they do)
```

Comment:
why is this necessary? should we instead put this logic in `./tests/macos/xcodeproj/build.sh` so it can be used when running locally?

Reply:
new test case will fail if changes are not in :)